src/extension/tests/helpers/RuntimeComposer.py

```diff
@@ -1,13 +1,16 @@
 import os
 import time
+
+from extension.src.TelemetryWriter import TelemetryWriter
 from extension.src.Utility import Utility
 from extension.src.file_handlers.JsonFileHandler import JsonFileHandler
 from extension.src.local_loggers.Logger import Logger
 
 
 class RuntimeComposer(object):
     def __init__(self):
-        self.logger = Logger()
+        telemetry_writer = TelemetryWriter()
+        self.logger = Logger(telemetry_writer=telemetry_writer)
```

Comment:
Note while reviewing; may remove as review completes -- ensure logger can be initialized without telemetry_writer. 
Regardless, telemetry_writer being on/off should be a configuration setting inside Constants.

Reply:
Confirmed: logger can be initialized without t_w

Reply:
I don't see the constant to turn telemetry_writer on and off? Expectation is it should feed into config factory into the t_w initialization (arming to on or off). With the possibility to be set to off with either that or when the Current env = Constants.Dev (not mandatory). 

Also please update open source contributor instructions with this to explain how to turn it off.

Reply:
Handled in PR #56 for core and PR #55 in handler

Reply:
follow up on implementation on #56